.circleci/config.yml

```diff
@@ -1043,12 +1053,10 @@ defaults: &defaults
         branch_pattern: "master"
 deploy_defaults: &deploy_defaults
   filters:
-    branches:
-      only:
-        - master
-        - stage
     tags:
       only: *tag_regex
+    branches:
+      ignore: /.*/
```

Comment:
Doesn't this prevent all deploy runs from running? Or does the `tags` above override it?

Reply:
I don't fully understand it either. Apparently you have to have this if you want the step to run only on a tag. Obviously I can't test this until we actually tag!